src/bitbucket/BitbucketAPI.ts

```diff
@@ -1,6 +1,6 @@
-import axios, { AxiosError } from 'axios';
+import axios, { AxiosResponse } from 'axios';
 import * as jwtTools from 'atlassian-jwt';
-import * as pRetry from 'p-retry';
+import delay from 'delay';
```

Comment:
Can you elaborate a bit this change of lib?

Reply:
Have replaced `p-retry` with doing it manually because I tried to reduce the cases in which we retry the API, and `p-retry` has semi-opinionated so was easier to do it manually.

`delay` is just used to poll the merge task every 3 seconds.